guia/closures.md

```diff
@@ -1 +1,131 @@
+
+*Closures* podem capturar e armazenar referências para quaisquer constantes e variáveis do contexto no qual ele foi definido. Isso é conhecido como *closure* sobre constantes e variáveis. O Swift se encarrega de realizar o gerenciamento de memória de captura por você.
+
+>Nota
+
+>Não se preocupe se você não esta familiarizado com o conceito de captura. Isso é explicado em detalhes adiante em [Capturando Valores](./closures.md#capturing_values).
+
+Funcões globais e alinhadas, como apresentado em [Funcões](./functions.md#functions), são casos realmente especiais de *closures*. *Closures* possuem uma, de três formas:
+
+* Funcões globais são *closures* que possuem um nome e podem capturar quaisquer valores.
+* Funções alinhadas são *closures* que possuem um nome e podem capturar valores a partir de suas funções fechadas.
+* Expressões de *closure* são *closures* sem nome, escritos com uma sintaxe limpa, que podem capturar valores dos contextos ao seu redor.
+
+As expressões de *closure* do Swift possuem uma sintaxe clara com otimizações que encorajam uma sintaxe curta e que evita mal entendidos em cenários comums. Essas otimizações incluem:
+
+* Dedução de parametros e tipos de valor de retorno a partir do contexto.
+* Retorno implícito a partir de expressões de *closure* únicas.
+* Argumentos com nomes simples .
+* Sintaxe do *closure* de fuga. - do inglês *Trailing closure sintax*.
+
+###Expressões de ***closure***
+
+Funções alinhadas, como apresentado em [Funções Alinhadas](./functions.md#nested_functions), são um meio conveniente de nomear e definir blocos independentes de código como parte de uma função maior. Contudo, algumas vezes é útil escrever versões mais curtas de funções como contructos sem uma declaração completa e nome. Isso é particularmente verdade quando você trabalha com funções ou métodos que pegam funções como um ou mais de seus parametros.
+
+Expressões de *closure* são uma meio de escrever *closures* com uma sintax breve e clara. Expressões de *closure* proporcionam varias otimizações de sintaxe para se escrever *closures* de uma maneira breve sem que haja perca de claridade ou intenção. Os exemplos de expressões de *closure* abaixo ilustram essas otimizações através do refinamento de um único exemplo de um methodo de ordenação `sort(_:)` através de várias iterações, cada qual expressa a mesma funcionalidade na maneira mais sucinta.
+
+####O Metódo de Ordenação
+
+A biblioteca padrão do Swift fornece um metódo chamado `sort(_:)`, que ordena um `array` de valores com tipo conhecido, baseado na saída de um *closure* de ordenação que você fornece. Uma vez que o seu *closure* complete o processo de ordenação, o método `sort(_:)` devolve um novo `array` do mesmo tipo e tamanho do original, com os elementos ordenados corretamente. O `array` original não é modificado pelo método `sort(_:)`.
```

Comment:
Aqui **array** não precisa estar entre crases, dado que não está fazendo referência ao tipo `Array`.

Acho que ou pode colocar em itálico, ou usar simplesmente a palavra **lista**.

cc @talesp